opencraft/settings.py

```diff
@@ -36,6 +36,7 @@
 
 env = environ.Env()
 root = environ.Path(os.path.dirname(__file__), os.pardir)
+environ.Env.read_env('.env')
```

Comment:
@brousch Why is this needed? It should already be done by `environ.Env()`, no? Were you getting issues without it?